api/v2/openapi.yaml

```diff
@@ -120,6 +120,23 @@ paths:
         '400':
           description: Malformed request
 
+  '/users/self/locations/types':
+    get:
+      description: List all locationtypes.
+      tags:
+        - locationtypes
```

Comment:
Dieser Tag wird als Überschrift in swagger angezeigt, vielleicht wäre dann `Locations` oder `Location Types` schöner.